alert/views.py

```diff
@@ -129,9 +130,9 @@ def accept_webhook(request):
 
     if username != settings.WEBHOOK_USERNAME or \
             password != settings.WEBHOOK_PASSWORD:
-        return HttpResponse('''Your credentials cannot be verified. 
-        They should be placed in the header as &quot;Authorization-Bearer&quot;,  
-        YOUR_APP_ID and &quot;Authorization-Token&quot; , YOUR_TOKEN"''', status=401)
+        return HttpResponse("""Your credentials cannot be verified.
```

Comment:
Should be a separate PR but need a `TODO` to change this error message, since we're actually using basic auth for the webhook